src/main/java/build/buf/protovalidate/internal/celext/CustomOverload.java

```diff
@@ -484,7 +535,7 @@ private static boolean validateEmail(String addr) {
     try {
       InternetAddress emailAddr = new InternetAddress(addr);
       emailAddr.validate();
-      if (addr.contains("<")) {
+      if (addr.contains("<") || !emailAddr.getAddress().equals(addr)) {
```

Comment:
Upstream Go implementation got this fix for failing on whitespace in email addresses.